pallets/ddc-staking/src/lib.rs

```diff
@@ -861,6 +866,46 @@ pub mod pallet {
 			<Pricing<T>>::set(Some(price_per_byte));
 			Ok(())
 		}
+
+		/// Add a new account to the list of cluster managers.
+		///
+		/// RuntimeOrigin must be Root to call this function.
+		#[pallet::weight(T::WeightInfo::allow_cluster_manager())]
+		pub fn allow_cluster_manager(
+			origin: OriginFor<T>,
+			grantee: <T::Lookup as StaticLookup>::Source,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let grantee = T::Lookup::lookup(grantee)?;
+			ClusterManagers::<T>::mutate(|grantees| {
+				if !grantees.contains(&grantee) {
+					grantees.push(grantee);
+				}
+			});
+
+			Ok(())
+		}
+
+		/// Remove an account from the list of cluster managers.
+		///
+		/// RuntimeOrigin must be Root to call this function.
+		#[pallet::weight(T::WeightInfo::disallow_cluster_manager())]
+		pub fn disallow_cluster_manager(
+			origin: OriginFor<T>,
+			revokee: <T::Lookup as StaticLookup>::Source,
+		) -> DispatchResult {
+			ensure_root(origin)?;
+
+			let revokee = T::Lookup::lookup(revokee)?;
+			ClusterManagers::<T>::mutate(|grantees| {
+				if let Some(pos) = grantees.iter().position(|g| g == &revokee) {
+					grantees.remove(pos);
+				}
+			});
+
```

Comment:
let's add an event here and in tests as well

Reply:
Events not required by the user.

Reply:
Do you mean by cluster management?

Reply:
Yes

Reply:
The use does not require tracking live updates or time of the list modifications.